UnitsNet.Tests/CustomCode/EnergyTests.cs

```diff
@@ -36,17 +36,17 @@ public class EnergyTests : EnergyTestsBase
 
         protected override double CaloriesInOneJoule => 0.239005736;
 
-        protected override double DecaeuThermsInOneJoule => 9.4781339449889105832843629746176e-10;
+        protected override double DecathermsEcInOneJoule => 9.47816988e-10;
 
-        protected override double DecaimperialThermsInOneJoule => 9.478171203551087813109937767482e-15;
+        protected override double DecathermsImperialInOneJoule => 9.478171203551087813109937767482e-15;
 
-        protected override double DecausThermsInOneJoule => 9.4804342797334860315281322406817e-10;
+        protected override double DecathermsUsInOneJoule => 9.4804342797334860315281322406817e-10;
 
         protected override double ElectronVoltsInOneJoule => 6.241509343260179e18;
 
         protected override double ErgsInOneJoule => 10000000;
 
-        protected override double EuThermsInOneJoule => 9.4781339449889105832843629746176e-9;
+        protected override double ThermsEcInOneJoule => 9.47816988e-9;
```

Comment:
Any reason you shortened this?
Also I get a slightly different number [here](http://www.metric-conversions.org/energy-and-power/joules-to-therms-ec.htm): `1J= 9.4781712e-9Therm(EC)`

Reply:
@eriove Mentioned this is probably more correct value. Please see the screenshot below.

![2017-09-26_1219](https://user-images.githubusercontent.com/31775071/30852654-2dd5201e-a2b5-11e7-8afd-961987151736.png)

Reply:
Oh, okay. I have no familiarity with these units, why is there a different number in the calculator? Are there different standards for EC therms?

Second converter here:
http://www.kylesconverter.com/energy,-work,-and-heat/joules-to-therms-(e.c.)
`9.478171203e-9`

I think @eriove possibly found a value that was not entirely accurate.

From the last link:
> 1 Therm (E.C.):
Exactly 100,000 BTUIT. Approximately 1.05505585262 x 108 Joules (SI). Therm (E.C.) is based on the international table BTU, see Therm (U.S.) for Therms based on 59 °F BTU.

Reply:
I took the number from EC Therms on the [Wikipedia page](https://en.wikipedia.org/wiki/Therm#Definitions) which links to the ISO BTU, but I have no familiarity either so @DInozemtsev have to decide.

Reply:
Ok, I can explain how I got a previous value.
So according Wikipedia:
`1 Th(E.C.) ≡ 100,000 BTU = 105,505,600 joules`

To get 1 Joule in Therms (E.C.):

`1/105,505,600 = 9.4781698791343777012784155532976e-9`
 (this is what I get at Windows 10 Calculator.)

I think we should go with this value. @angularsen what do you think about this?

Reply:
I think your value 105,505,600 is a rounded value, please read the quote from wiki:
> 1 Therm (E.C.):
Exactly 100,000 BTUIT. Approximately 1.05505585262 x 108 Joules (SI). Therm (E.C.) is based on the international table BTU, see Therm (U.S.) for Therms based on 59 °F BTU

`1 Th EC = 1e5 BTU` exactly and approx `105,505,852.62 joules`. Then doing `1/x` gives me `9.4781712031331720001278504447561e-9`, which I believe is the correct value as it also matches the two online converters in above comments. This value should be used in the test case. Also since this is an infinite value, I think we should use the expression `x/105505585.262` in the JSON for `FromBaseToUnitFunc`, and `x*105505585.262` for `FromUnitToBaseFunc`.